.github/dependabot.yml

```diff
@@ -8,3 +8,10 @@ updates:
       interval: "monthly"
     commit-message:
       prefix: "ci(dependabot):"
+
+  - package-ecosystem: "pip"
+    directory: "/resources"
+    schedule:
+      interval: "daily"
```

Comment:
I think monthly is a nicer experience here?

Reply:
I prefer fast information about problems. Maybe weekly. But this is a weak opinion.

Reply:
I'll vote for `weekly`. `daily` is fast enough that I would ignore it, but a month could be a long time to not know about a vulnerability.

Reply:
My personal opinion is that we have a very high volume of issues and PRs, and we want to reduce that as much as possible. Since the dependencies are pinned, I don't see a huge advantage in getting notifications early. Even yearly would be fine. However, I'm generally unlikely to be the one dealing with these so I defer to those that will. 🤷 My opinion is also weak, hence why I didn't block or anything — people should feel free to merge this when ready! 😊